<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21_MainWindow</class>
 <widget class="QMainWindow" name="s21_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>s21_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: rgb(42, 42, 42);
}</string>
   </property>
   <widget class="QPushButton" name="Two_Button">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ymax_Label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>35</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: white;
}</string>
    </property>
    <property name="text">
     <string>Ymax =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Six_Button">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>193</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="X_Button">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>382</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Atan_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>256</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QLabel" name="Xmin_Label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>5</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Xmin =</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Ymax">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
color: white;
}</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="LR_Button">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>67</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Div_Button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(230, 143, 52);
	color: white;
	border: 1px solid white;
	font-size: 18px;
	font-weight: 700;
}

QPushButton:pressed {
	background-color: rgb(119, 118, 123);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Graph_Button">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>382</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nine_Button">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>256</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Del_Button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>67</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mod_Button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>67</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Xmin">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
color: white;
}</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QCustomPlot" name="Graph" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>69</y>
      <width>420</width>
      <height>310</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Log_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>382</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dot_Button">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>319</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AC_Button">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>67</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Seven_Button">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>256</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QLabel" name="Calculation_label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	qproperty-alignment: 'AlignVCenter | AlignRight';
	border-bottom: 1px solid-gray;
	color: white;
}
background-color: white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LP_Button">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>67</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eq_Button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>383</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(230, 143, 52);
	color: white;
	border: 1px solid white;
	font-size: 18px;
	font-weight: 700;
}

QPushButton:pressed {
	background-color: rgb(119, 118, 123);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="Xmax_Label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>35</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: white;
}</string>
    </property>
    <property name="text">
     <string>Xmax =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Three_Button">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Mult_Button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>193</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(230, 143, 52);
	color: white;
	border: 1px solid white;
	font-size: 18px;
	font-weight: 700;
}

QPushButton:pressed {
	background-color: rgb(119, 118, 123);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Zero_Button">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>319</y>
      <width>122</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Five_Button">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>193</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Pow_Button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>319</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Ymin">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
color: white;
}</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="Four_Button">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>193</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sin_Button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eight_Button">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>256</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QLabel" name="Ymin_Label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>5</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: white;
}</string>
    </property>
    <property name="text">
     <string>Ymin =</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Acos_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>193</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_Button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>319</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(230, 143, 52);
	color: white;
	border: 1px solid white;
	font-size: 18px;
	font-weight: 700;
}

QPushButton:pressed {
	background-color: rgb(119, 118, 123);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="One_Button">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(130, 131, 133);
	color: white;
	border: 1px solid white;
	font-size: 20px;
	font-weight: 600;
}
QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Tan_Button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>256</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sub_Button">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>256</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(230, 143, 52);
	color: white;
	border: 1px solid white;
	font-size: 18px;
	font-weight: 700;
}

QPushButton:pressed {
	background-color: rgb(119, 118, 123);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Xmax">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
color: white;
}</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="Cos_Button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>193</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Sqrt_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>67</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Credit_Button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>382</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>credit</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Double_Spin_Box">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>382</y>
      <width>120</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox {
color: white;
}</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="Ln_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>319</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Asin_Button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="E_Button">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>382</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(59, 60, 62);
	color: white;
	border: 1px solid white;
	font-size: 18px;
    font-weight: 600;
}

QPushButton:pressed {
	background-color: rgb(230, 143, 52);
}</string>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>third_party/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
